frontend-admin/src/app/views/customers/customers.controller.coffee

```diff
@@ -1,4 +1,4 @@
-@App.controller 'CustomersController', ($uibModal, MnoeUsers, MnoeOrganizations, MnoeObservables, OBS_KEYS) ->
+@App.controller 'CustomersController', ($uibModal, $scope, MnoeUsers, MnoeOrganizations, MnoeObservables, OBS_KEYS) ->
```

Comment:
Plese use this order: `$scope, $uibModal, MnoeUsers, MnoeOrganizations, MnoeObservables, OBS_KEYS`